docs/agile-process.md

```diff
@@ -16,7 +16,7 @@ There is a [high-level roadmap](https://github.com/GSA-TTS/FAC/blob/main/docs/pr
 **Labeling issues** - In addition to the "story", "design" and "engineering" labels, the "needs refinement" label is a very important label to denote whether an issue needs to be better clarified. There are a bunch of other labels that are rarely used and could probably be deleted. These two carry special significance since we try not to actively work on an issue until these labels have been removed: 
 
 * “Needs refinement” - need a design, engineering or product decision to move forward
-* “Needs scoping” - needs people to pair up and review, then break down an issue into individual tasks.
+* “Needs scoping” - needs people to pair up and review, then break down an issue into individual tasks and estimate them using beverage sizes. Assuming this is an engineering story, the will put another beverage size on the story level which will be used to determine capacity though an eyeball, non-mathematical judgement. 
```

Comment:
minor typos:

"the will put" should be "they will put"
"though an eyeball" should be "through an eyeball"

Reply:
Thanks Tim! Updated.